### DISCUSSION TITLE
response_model been created twice.

### DISCUSSION THREAD

**Discussion by Chise1** [fastapi/fastapi#6700] [Questions]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorial in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:

### Example
When I return a BaseModel(It is the router's response_model),the Model has been created once again.

```Python
from fastapi import FastAPI
from pydantic import BaseModel


class T(BaseModel):
    id: int

    def __init__(self, **kwargs):
        super().__init__(**kwargs)
        print("1")

app = FastAPI()


@app.get("/",response_model=T)
def read_root():
    return T
```
The "1" will been print twice.
Maybe need instance the return ,if instance(r,T) is true,return the r . 
### Description

* Open the browser and call the endpoint `/`.
* It returns a JSON with `{"Hello": "World"}`.
* But I expected it to return `{"Hello": "Sara"}`.

### Environment
* OS Linux
* FastAPI Version ^0.63
To know the FastAPI version use:

**Comment by adriangb** [comment]
I wonder if this could be https://github.com/samuelcolvin/pydantic/issues/1491?
